dnf-behave-tests/features-createrepo_c/createrepo_c_cache.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Tests createrepo_c --cache option
+
+
+Scenario: create and use cache files
+Given I create directory "/temp-repo/"
+Given I copy file "{context.dnf.repos_location}/createrepo_c-ci-notinstallable/x86_64/createrepo_c-0.14.3-1.fc29.x86_64.rpm" to "/temp-repo"
+Given I create symlink from "{context.dnf.repos_location}/createrepo_c-ci-notinstallable/x86_64/createrepo_c-devel-0.14.3-1.fc29.x86_64.rpm" to "/temp-repo"
+Given I copy file "{context.dnf.repos_location}/createrepo_c-ci-notinstallable/x86_64/createrepo_c-libs-0.14.3-1.fc29.x86_64.rpm" to "/temp-repo"
+Given I create directory "/temp-repo/cache"
+ When I execute createrepo_c with args "--cachedir ./cache ." in directory "/temp-repo"
+ Then the exit code is 0
+  And repodata "/temp-repo/repodata/" are consistent
+  And repodata in "/temp-repo/repodata/" is
+     | Type                | File                                | Checksum Type | Compression Type |
+     | primary             | ${checksum}-primary.xml.gz          | sha256        | gz               |
+     | filelists           | ${checksum}-filelists.xml.gz        | sha256        | gz               |
+     | other               | ${checksum}-other.xml.gz            | sha256        | gz               |
+     | primary_db          | ${checksum}-primary.sqlite.bz2      | sha256        | bz2              |
+     | filelists_db        | ${checksum}-filelists.sqlite.bz2    | sha256        | bz2              |
+     | other_db            | ${checksum}-other.sqlite.bz2        | sha256        | bz2              |
+ And primary in "/temp-repo/repodata" contains only packages
```

Comment:
Indentation is another thing, I'm really not a fan of this style of alignment exactly because this iadvertedly will start happening - this `And` is one space off to the left, as well as the tables above and below it. This also repeats throughout the whole commit.

Please align the tables as well as string literals to the six space indent and fix the `And`s that are off... It probably wouldn't be hard to come up with a sed command to fix the majority of it.